WatchFaceAlphaKotlin/app/src/main/res/layout/watch_face_and_complications_preview.xml

```diff
@@ -16,21 +16,51 @@
   -->
 <!--
     ImageView to load preview of watch face.
-    TODO (codingjeremy): Add mask for screenshot and buttons to enable editing of complications.
 -->
-<RelativeLayout
+<androidx.constraintlayout.widget.ConstraintLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="wrap_content"
-    android:layout_height="wrap_content">
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/analog_complication_settings_preview_size"
+    android:layout_height="@dimen/analog_complication_settings_preview_size"
+    tools:context=".editor.WatchFaceConfigActivity">
+
+    <Button
+        android:id="@+id/left_complication"
+        android:layout_width="0dp"
+        android:layout_height="match_parent"
```

Comment:
Small nit: we should avoid `match_parent` in `ConstraintLayout` children, and instead use `0dp`:

https://developer.android.com/training/constraint-layout#adjust-the-view-size

Reply:
Good call, I replaced all of them.